os/include/tinyara/fs/ioctl.h

```diff
@@ -451,6 +452,19 @@
 #define COMPIOC_GET_COMP_NAME      _COMPIOC(0x0003)
 #define COMPIOC_DECOMPRESS	   _COMPIOC(0x0004)
 
+/* Pm driver ioctl definitions *****************************/
+#define _PMIOCVALID(c)      (_IOC_TYPE(c) == _PMBASE)
+#define _PMIOC(nr)          _IOC(_PMBASE, nr)
+
+#define PMIOC_LOCK               _PMIOC(0x0001)
+#define PMIOC_UNLOCK             _PMIOC(0x0002)
+#define PMIOC_TIMER_CREATE       _PMIOC(0x0003)
```

Comment:
i have assumed whenever an app wants to use a timer, it will have to first use TIMER_CREATE to create a timer. 
Then to set the timer, app need to use TIMER_SET just after executing its required function. For the case of wifi sending, after sending wifi , it should always do TIMER_SET. 
TIMER_STOP will be used to stop the timer temporarily.
TIMER_DELETE will be used to completely remove the timer from app , so that other app can use it.